app/javascript/dashboard/assets/scss/_layout.scss

```diff
@@ -9,87 +9,25 @@ body {
 }
 
 .app-wrapper {
-  @include full-height;
-  flex-grow: 0;
-  min-height: 0;
-  width: 100%;
+  @apply h-full flex-grow-0 min-h-0 w-full;
```

Comment:
@scmmishra Do you mean moving this `h-full flex-grow-0 min-h-0 w-full` to inline one?

Reply:
@scmmishra At the moment, we're using the `app-wrapper` class in the super admin. Just want to let you know that if we move it, it might cause some issues.

Reply:
We can keep the style, just move the top-level (base) style for now, the rest can be migrated later

So the new class list for the component would be `app-wrapper h-full flex-grow-0 min-h-0 w-full`

Reply:
Done